lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -351,3 +359,201 @@ def _load_platform(
 
     platform.initialize()
     return platform
+
+
+@dataclass_json
+@dataclass
+class SigTransformerSchema(schema.Transformer):
+    # raw vhd URL, it can be the blob under the same subscription of SIG
+    # or SASURL
+    vhd: str = field(default="", metadata=field_metadata(required=True))
+    # if not specify gallery_resource_group_name, use shared resource group name
+    gallery_resource_group_name: str = field(default=AZURE_SHARED_RG_NAME)
+    # if not specified, will use the first location of gallery image
+    gallery_resource_group_location: str = field(default="")
+    # gallery name, it can be reused if this gallery exists
+    gallery_name: str = field(default="lsg_gallery_test")
```

Comment:
This is not a good default name, how about "lisa_default_gallery"?

Reply:
updated